dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -311,6 +311,17 @@
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 		};
+
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
+			interrupts = <9 0 10 0 10 0 11 0 11 0>;
```

Comment:
Is it correct priority==0?
Moreover, how should I read the sequence of couples?

Reply:
Priority is 0 means using the default priority I think. Sorry I don't understand what is 'sequence of couples', Could you give me a more specific description?

Reply:
Like 9 0, 10 0, 10 0, 11 0, 11 0
Five couples which represent, I guess, irqs for the DMA channel/slot.
Can you describe it to me better? Thx

Reply:
> Like 9 0, 10 0, 10 0, 11 0, 11 0
> Five couples which represent, I guess, irqs for the DMA channel/slot.
> Can you describe it to me better? Thx

Sorry for the late reply. For five couples, the first one in them represents the irq channel used in hardware, which is described in the datasheet, and the second one is the priority we would use. This DMA instance has 5 channels in hardware, so five couples of data. This is the default explanation for 'interrupts' for all dts nodes. Hope this anwsers your question.